poetry.lock

```diff
@@ -252,6 +263,7 @@ version = "1.23.0"
 description = "Microsoft Azure Identity Library for Python"
 optional = false
 python-versions = ">=3.9"
+groups = ["main"]
```

Comment:
Do you know why this is suddenly being added?

Reply:
its done automatically by dependabot